src/main/scala/com/raphtory/core/storage/pojograph/PojoGraphLens.scala

```diff
@@ -103,6 +105,7 @@ final case class PojoGraphLens(
   def runGraphFunction(f: Vertex => Unit): Unit = {
     vertices.foreach { case (id, vertex) => f(vertex) }
     vertexCount.set(vertices.size)
+    StorageTelemetry.pojoLensVertexCount.inc()
```

Comment:
I think with this being outside of the foreach it is just counting the number of functions run? What do you want it to count?